### ISSUE TITLE
Prefill input fields with url parameters

### ISSUE THREAD

**Issue by ImaCrea** [SergeyMosin/Appointments#145]
Hi there,
thank you very much for this great app. 

I was wondering if there was a way to **prefill some input fields through parameters set in URL**?

For instance, apps/appointments/pub/_RWu%2Be%2B6rpbWsQ%3D%3D/form?name=cat&mail=cat@doe.com would load the form with appropriate fields already filled.

**Comment by SergeyMosin** [comment]
Good idea.

**Comment by nielosz** [comment]
I've done this with Javascript loader app and 

```
// Prefill forms, pattern prfl_<input-id>=value
(new URL(window.location.href))
  .searchParams.forEach((x, y) =>{
      if (y.indexOf('prfl_') == 0) 
        document.getElementById(y.substring(5)).value = x;
    });

```

**Comment by rgon** [comment]
@SergeyMosin I have implemented this feature in PR #537

For ease-of-use purposes, it follows the same standard as cal.com

```
https://mycloud.tld/appointments/appointmentID?name=UserName&email=userEmail&hidePrefilledInputs=true
```

**Comment by SergeyMosin** [comment]
V2.1.9

`Settings > Form Settings > Advanced Form Settings > Allow Prefilled Inputs | Prefilled Inputs Appearance`

_Next step is to add auto-prefill for logged-in users as per #56_

----
![image](https://github.com/user-attachments/assets/2aa5ac6c-663e-4ec7-826d-95bedbbfcd96)
